inc/consent/namespace.php

```diff
@@ -64,8 +66,10 @@ function set_consent_options() {
 
 	// If any options were set in the config, update those values in the database and remove the options on the settings page.
 	if ( ! empty( $options ) ) {
-		update_option( 'cookie_consent_option', $options );
-		add_filter( 'altis.consent.consent_settings_fields', $fields );
+		add_filter( 'pre_option_cookie_consent_options', function () use ( $options ) {
+			return $options;
+		} );
+		add_filter( 'altis.consent.consent_settings_fields', '__return_empty_array' );
```

Comment:
Is this right? Some options might be configured but not all I thought

Reply:
Not sure, was just fixing the implementation bug, not necessarily the behaviour. Can be handled in #92 as desired.